internal/factory/workflow_helpers.go

```
@@ -59,3 +59,25 @@ func setInvocationSurface(invocation *store.Invocation, surface terminal.Surface
+func workerIDForInvocation(invocation store.Invocation) string {
+	definition, err := roleDefinitionForInvocation(invocation)
+	if err == nil && definition.Kind == workflow.RoleKindReview {
+		return invocation.RunID + "-" + invocation.ID
+	}
+	return invocation.RunID
+}
+
+// workerIDForRole returns the worker identity used before an invocation is
+// persisted. Review callers should prefer workerIDForInvocation once the
+// invocation ID exists so the worker resources are fresh per iteration.
+func workerIDForRole(runID, role string) string {
```

golangci-lint (2.12.2)

[error] func workerIDForRole is unused

(unused)